definitions/npm/express_v4.x.x/flow_v0.32.x-/express_v4.x.x.js

```diff
@@ -40,6 +44,7 @@ declare class express$Request extends http$IncomingMessage mixins express$Reques
   header(field: string): string | void;
   is(type: string): boolean;
   param(name: string, defaultValue?: string): string | void;
+  accepts(types: Array<string>): string | false;
```

Comment:
:+1: One suggestion: move it up to line 41, so it's near the other `accepts` variant.